marlowe-playground-client/test/Marlowe/ContractTests.purs

```diff
@@ -37,15 +37,15 @@ all =
 
         (Tuple _ finalState) =
           (flip runState mkState) do
-            updateMarloweState (set _executionState (emptyExecutionStateWithSlot zero))
+            updateMarloweState (set _executionState (emptyExecutionStateWithSlot zero Close))
```

Comment:
Being on the final step it makes sense that the contract is `Close`, but I wonder if this is a signal that we shouldn't keep the Contract per simulation step, and have a single global version of the Contract.

Maybe this is a refactor to do later on, if we decide to show the full contract with markers indicating "the current path"

Reply:
Ah, no, this was just me being lazy. I would pass `Contracts.escrow` here but, because there is a call to `updateContractInState` afterwards that overrides this, I am not sure whether that call does something extra, so I just left it there and put `Close` here as a filler. Maybe I should try removing it and see if it still works

Reply:
Oh, yes, it was just because `updateContractInState` does the parsing, and I didn't want to embed there the code for parsing, which is for Extended Marlowe and we need Core Marlowe, so it just makes things complicated. We may want a separate parser for Core Marlowe eventually...